<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QMainWindow" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibrationWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="config_Polaris_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Polaris</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_ATC_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>ATC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capture_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cal1_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cal2_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
